flash/core/model.py

```diff
@@ -156,8 +160,13 @@ def predict(
         output = data_pipeline.uncollate_fn(predictions)  # TODO: pass batch and x
         return output
 
-    def configure_optimizers(self) -> torch.optim.Optimizer:
-        return self.optimizer_cls(filter(lambda p: p.requires_grad, self.parameters()), lr=self.learning_rate)
+    def configure_optimizers(
+            self
+    ) -> Union[Tuple[Tuple[torch.optim.Optimizer], SchedulerType], Tuple[torch.optim.Optimizer]]:
+        optimizers = self.optimizer_cls(filter(lambda p: p.requires_grad, self.parameters()), lr=self.learning_rate),
+        if self.schedulers:
```

Comment:
What happens if the scheduler needs to wrap the optimizer(s) ? Any solution for it ?

Reply:
TBH I am not sure about this. In LightningModule either we return "list of optimizer and scheduler" or only optimizer. So, my thinking was that we can do the same here. @tchaton

Reply:
it would be helpful for me if you can provide some reference for scheduler wrapping the optimizer.

Reply:
```suggestion
        if self.scheduler:
```